Content.Server/_Umbra/Atmos/Components/SpontaneousCombustionProtectionComponent.cs

```diff
@@ -0,0 +1,20 @@
+
+using Content.Server.Atmos.EntitySystems;
+
+namespace Content.Server.Atmos.Components;
```

Comment:
```suggestion
namespace Content.Server._Umbra.Atmos.Components;
```
This shouldn't cause issues and is more accurate.

Reply:
The namespaces were fixed.